Content.IntegrationTests/Tests/Weapons/WeaponSniperTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Content.IntegrationTests.Tests.Interaction;
+using Content.Shared.Damage;
+using Content.Shared.Weapons.Ranged.Components;
+using Content.Shared.Weapons.Ranged.Events;
+using Content.Shared.Wieldable.Components;
+using Robust.Shared.Prototypes;
+
+namespace Content.IntegrationTests.Tests.Weapons;
+
+public sealed class WeaponSniperTests : InteractionTest
+{
+    private static readonly EntProtoId MobHuman = "MobHuman";
+    private static readonly EntProtoId SniperMosin = "WeaponSniperMosin";
+
+    [Test]
+    public async Task SniperMosinInteractionTest()
+    {
+        var urist = await SpawnTarget(MobHuman);
+        var damageComp = Comp<DamageableComponent>(urist);
+        var startDamage = damageComp.TotalDamage.Value;
+
+        var mosinNet = await PlaceInHands(SniperMosin);
+        var mosinEnt = SEntMan.GetEntity(mosinNet);
```

Comment:
```suggestion
        var mosinEnt = ToServer(mosinNet);
```
you can use the shorthand for this

Reply:
I didn't know that was what that was for. Neat. I was under the impression that was like dirty.